src/components/StyledReactSelect/StyledReactSelect.tsx

```diff
@@ -0,0 +1,52 @@
+import { ComponentProps, useMemo } from 'react'
+import Select, { components, OptionProps } from 'react-select'
+import { ChevronDownIcon, ThickCheckIcon } from '@radix-ui/themes'
+import { getStylesConfig, getThemeConfig } from './StyledReactSelect.styles'
+
+export function StyledReactSelect<Option>(
```

Comment:
Far from ideal integration of ReactSelect into Radix, it doesn't support all the props Radix's select does and has static size, but this iteration I've focused on blending it into Radix styles.